function/templates/TriggerAuthentication.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.triggerAuthName }}
+apiVersion: keda.k8s.io/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: {{- .Values.triggerAuthName }}
+spec:
+  podIdentity:
+    provider: {{- .Values.trigger.podIdentityProvider }} # none | azure | gcp | spiffe | aws-eks | aws-kiam
```

Comment:
```suggestion
    provider: {{ .Values.trigger.triggerAuthName }} # none | azure | gcp | spiffe | aws-eks | aws-kiam
```

Reply:
This needs to be "azure" for the provider rather than the resource name so i think it's correct as is?

Reply:
sorry, i meant it doesn't match they key (triggerPodIdentity)  in values.yaml https://github.com/hmcts/chart-function/pull/8/files#diff-f4ea29544a9020b6afe470febd24ef18afc163a3477969b49fe227bfa166fb43R27